sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample9_DifferentiateOutputModelsTrainedWithAndWithoutLabels.cs

```diff
@@ -0,0 +1,118 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Azure.AI.FormRecognizer.Models;
+using Azure.AI.FormRecognizer.Tests;
+using Azure.AI.FormRecognizer.Training;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.FormRecognizer.Samples
+{
+    public partial class FormRecognizerSamples : SamplesBase<FormRecognizerTestEnvironment>
+    {
+        /// This sample demonstrates the differences in output that arise when StartRecognizeCustomForms
+        /// is called with custom models trained with labels and without labels.
+
+        [Test]
+        public async Task OutputModelsTrainedWithLabels()
+        {
+            string endpoint = TestEnvironment.Endpoint;
+            string apiKey = TestEnvironment.ApiKey;
+            string trainingFileUrl = TestEnvironment.BlobContainerSasUrl;
+            string formFilePath = FormRecognizerTestEnvironment.CreatePath("Form_1.jpg");
+
+            FormTrainingClient trainingClient = new FormTrainingClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+            FormRecognizerClient client = new FormRecognizerClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+
+            // Model trained with labels
+            CustomFormModel modelWithLabels = await trainingClient.StartTraining(new Uri(trainingFileUrl), useTrainingLabels: true).WaitForCompletionAsync();
+
+            using (FileStream stream = new FileStream(formFilePath, FileMode.Open))
+            {
+                RecognizedFormCollection formsWithLabels = await client.StartRecognizeCustomForms(modelWithLabels.ModelId, stream).WaitForCompletionAsync();
+
+                // With a form recognized by a model trained with labels, the 'field.Name' key will be its
+                // training-time label. Note that Label data is not returned for model trained with labels.
+                Console.WriteLine("---------Recognizing forms with models trained with labels---------");
+                foreach (RecognizedForm form in formsWithLabels)
+                {
+                    Console.WriteLine($"Form of type: {form.FormType}");
+                    foreach (FormField field in form.Fields.Values)
+                    {
+                        Console.WriteLine($"Field {field.Name}: ");
+                        Console.WriteLine($"    Value: '{field.ValueText.Text}");
+                        Console.WriteLine($"    Confidence: '{field.Confidence}");
+                    }
+                }
+
+                // Find a specific labeled field.
+                // For this particular sample, we will look for the known training-time label 'VendorName'.
```

Comment:
IIRC searching within the results came up in UX feedback as well -- would it be possible to show a method that uses Linq with a StartsWith expression as well, just to show this capability?

Reply:
>IRC searching within the results came up in UX feedback as well 

Correct. This is why we are adding this section to both methods.

>Using Linq

What will be the advantage from what I currently have? both will execute a loop and doing a `TryGetValue` is close to O(1)

Reply:
You can search for something with an inexact name by writing something like:
```C#
form.Fields.Where(kv => kv.Key.StartsWith("Ven"));
form.Fields.Where(kv => kv.Key.Contains("Name"));
```

Reply:
updated